src/views/MorphicBarEditor.vue

```diff
@@ -480,11 +476,11 @@ export default {
       }
     },
     getMakeAButtons: function () {
-      let buttons = []
+      const buttons = []
       if (availableItems && availableItems.length > 0) {
         for (let i = 0; i < availableItems.length; i++) {
           if (availableItems[i].configuration.subkind) {
-            let item = availableItems[i]
+            const item = availableItems[i]
```

Comment:
Same here, empty constant that you are starting modifying right away, makes no sense.